### PR TITLE
Added RowCellUtil class to get cell value efficiently.

### PR DESCRIPTION
**Pull request by rupeshitpatekarq**
@sduskis , could you please review it.

### CODE REVIEW

**Comment by sduskis** [comment]
We need something a bit different.  The interface would look like this:

```
class CellUtils {
  public static byte[] getValue(Cell cell) { ... } 
}
```

It would basically do the same thing that you do in `RowCellUtil.getValueArray`, but would not require a `RowCell` object.

**Comment by rupeshitpatekarq** [comment]
@sduskis , Created CellUtils class.

File: bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/RowCell.java [hunk 1]
```diff
@@ -139,7 +141,7 @@ public long getSequenceId() {
   /** {@inheritDoc} */
   @Override
   public byte[] getValueArray() {
-    return this.valueArray;
+    return CellUtils.getValue(this, this.valueArray);
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/RowCell.java]: This should still be `this.valueArray`.  There is no reason for CellUtils here.

File: bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/CellUtils.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ * <p> Utility methods helpful for slinging {@link Cell} instances.CellUtils class to provide utilities related to RowCell class. </p>
+ * 
+ * @author rupeshit
+ *
+ */
+public class CellUtils {
+
+  public static byte[] getValue(Cell cell,byte[] valueArray) {
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/CellUtils.java]: There's no reason to have valueArray here.

File: bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/CellUtils.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * 
+ * @author rupeshit
+ *
+ */
+public class CellUtils {
+
+  public static byte[] getValue(Cell cell) {
+    if(cell.getValueLength() >= cell.getValueArray().length) {
+      return cell.getValueArray();
+    }else if(cell.getValueLength() < cell.getValueArray().length) {
```
Comment (sduskis) [inline bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/CellUtils.java]: Why not just "else"? 

Also, please add a space between `if` and `(`.  There should also be a space between `{` and `else`

**Comment by rupeshitpatekarq** [comment]
@sduskis , please have a look now.

**Pull request merged**